lib/app.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express')
+const app = express()
+const logSymbols = require('log-symbols')
+const log = require('fancy-log')
+const db = require('../db')
+const addToLibreNet6 = require('../sendConfig')
+
+app.use(express.json())
+
+app.post('/submit', (req, res)=> {
+    if (!req.body.apiKey) { 
+        res.json({ error: 'missing api key' })
+        return
+    };
+    //todo checkConfig(req.body) => bolean
+    
+    db.get(req.body.apiKey, (err, value) => {
+        if(err) {
+            //wait 2 seconds if the api key is invalid
+            setTimeout(()=> res.json({ error: 'wrong api key' }), 2000)
+            log.error(logSymbols.error, 'Wrong api key used: '+ req.body.apiKey)
+            return
+        }
+        try {
+            value = JSON.parse(value)
+        } catch(e) { /*Don't do anything*/ }
```

Comment:
This try catch statement looks a bit dangerous, or at least not explicit enough.
What are the possible types of `value` here?
It is ok to continue the execution even if an exception is raised?

Reply:
Solved in https://github.com/LibreRouterOrg/soporteremoto-tiers/pull/1/commits/7b48712a83617ce669d084410a56209fcd221179